.github/workflows/vrt.yml

```
@@ -0,0 +1,171 @@
+        else
+          echo "has_diff=false" >> $GITHUB_OUTPUT
+        fi
+    
+    - name: Comment PR with VRT results
+      if: steps.vrt-check.outputs.has_diff == 'true'
+      uses: actions/github-script@v7
+      with:
+        script: |
+          const fs = require('fs');
+          const path = require('path');
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)